### PR TITLE
mutate: csp error info added

### PR DESCRIPTION
**Pull request by axelbostrom**
If javascript isn't loaded a information text will show

### CODE REVIEW

File: plugin/mutate/data/dashboard.html [hunk 1]
```diff
@@ -9,6 +9,17 @@
...
 
     <title></title>
+    
+    <div id ="csp_error">
+      <h2>Something isn't right...</h2>
+      <p>To be able to read the mutation testing report you have to turn off CSP in Firefox.</p>
+      <ol>
+        <li>Copy and paste the following into the address bar in firefox: about:config.</li>
+        <li>Search for "CSP".</li>
+        <li>Change "security.OCSP.enabled" from true to false.</li>
```
Comment (joakim-brannstrom-work1) [inline plugin/mutate/data/dashboard.html:19]: spelling? OCSP -> CSP?

File: plugin/mutate/data/dashboard.html [hunk 2]
```diff
@@ -9,6 +9,17 @@
     <meta name="author" content="Joakim Brännström">
 
     <title></title>
+    
+    <div id ="csp_error">
```
Comment (joakim-brannstrom-work1) [inline plugin/mutate/data/dashboard.html:13]: the div tag should be inside the body tag?

File: plugin/mutate/data/dashboard.html [hunk 3]
```diff
@@ -12,6 +12,16 @@
   </head>
 
   <body>
+    <div id ="csp_error">
+      <h2>Something isn't right...</h2>
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/dashboard.html:16]: Maybe add what the problem is.
"Something isn't right... Javascript need to be enabled"

File: plugin/mutate/data/dashboard.html [hunk 4]
```diff
@@ -12,6 +12,16 @@
...
+      <p>To be able to read the mutation testing report you have to turn off CSP in Firefox.</p>
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/dashboard.html:17]: If you are using Firefox there may be a problem with Content Security Protection (CSP) with the web server that is used to publish the mutation report. To see the report correctly you need to turn off CSP. Follow these steps:

**Pull request merged**